README.md

```diff
@@ -307,6 +307,7 @@ Visiting `https://localhost:4567/` should show a Yoti Connect button
 * Activity Details
   * [X] Remember Me ID `remember_me_id`
   * [X] Parent Remember Me ID `parent_remember_me_id`
+  * [X] Receipt ID `receipt_id`
```

Comment:
Can we add timestamp to the README too 👍

Reply:
Added in 3df0c0c